gremlin-dotnet/src/pom.xml

```diff
@@ -89,6 +93,54 @@ limitations under the License.
                             <nugetAddEnabled>false</nugetAddEnabled>
                         </configuration>
                     </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-antrun-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>pack-dotnet-template</id>
+                                <phase>package</phase>
```

Comment:
Would it be possible to move the execution to the `deploy` phase? that way on `install` or prior phases, the build wouldn't be affected.

Also, it won't require mono on TravisCI or docker-based builds.

Reply:
i'd asked that question earlier

https://github.com/apache/tinkerpop/pull/867#issuecomment-390267672

@FlorianHockmann response follows that.